application-collabora-ui/src/main/resources/Collabora/Code/UI.xml

```diff
@@ -618,13 +618,29 @@
     &lt;/div&gt;
   &lt;/div&gt;
 #end
+#macro (checkIfEnabled $isEnabled)
```

Comment:
I would rather:
* extend ``CollaboraConfiguration`` with the ``isEnabled()`` method
* add a script service to expose the Collabora configuration ``$services.collabora.configuration.isEnabled()``
* use the script service instead of this Velocity macro

Reply:
Just a small comment. I couldn't make `$services.collabora.configuration` work, so I used $services.collaboraConfiguration`